irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -0,0 +1,227 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ametsuchi/impl/postgres_wsv_command.hpp"
+#include <iostream>
+
+namespace iroha {
+  namespace ametsuchi {
+
+    PostgresWsvCommand::PostgresWsvCommand(pqxx::nontransaction &transaction)
+        : transaction_(transaction) {}
+
+    bool PostgresWsvCommand::insertAccount(const model::Account &account) {
+      pqxx::binarystring master_key(account.master_key.data(),
+                                    account.master_key.size());
+      std::stringstream permissions;
+      permissions << account.permissions.add_signatory
```

Comment:
I'm not sure whether `model` has to know about binary string, since this is database-dependent.